examples/kmlFetchGood.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+<!--
+See https://github.com/AnalyticalGraphicsInc/cesium-google-earth-examples/blob/master/LICENSE.md
+
+Fenway Model was used from the publicly available Model on Sketchup
+https://3dwarehouse.sketchup.com/model.html?id=21eee70f917ce34af7a0c817ab72c49b
```

Comment:
I had read the TOS, but did not see any exclusions.  It's moot anyways as I didn't end up using the Fenway model as we don't support loading Models from sketchup via a KML export.